creator-node/test/audiusUsers.js

```diff
@@ -66,3 +85,91 @@ describe('test AudiusUsers', function () {
       .expect(200)
   })
 })
+
+// Below block uses actual ipfsClient (unlike first describe block), hence
+// another describe block for this purpose
+describe('tests /audius_users/metadata metadata upload with actual ipfsClient', function () {
+  let app, server, session, libsMock, ipfs
+
+  // Will need a '.' in front of storagePath to look at current dir
+  // a '/' will search the root dir
+  before(async () => {
+    let storagePath = config.get('storagePath')
+    if (storagePath.startsWith('/')) {
+      storagePath = '.' + storagePath
+      config.set('storagePath', storagePath)
+    }
+  })
+
+  beforeEach(async () => {
+    ipfs = ipfsClient.ipfs
+    libsMock = getLibsMock()
+
+    const appInfo = await getApp(ipfs, libsMock, BlacklistManager)
+    await BlacklistManager.blacklist(ipfs)
+
+    app = appInfo.app
+    server = appInfo.server
+    session = await createStarterCNodeUser()
+  })
+
+  afterEach(async () => {
+    sinon.restore()
+    await server.close()
+  })
+
+  it('should fail if metadatda is not found in request body', async function () {
+    const resp = await request(app)
+      .post('/audius_users/metadata')
+      .set('X-Session-ID', session)
+      .send({ dummy: 'data' })
+      .expect(500)
+
+    assert.deepStrictEqual(resp.body.error, 'Internal server error')
```

Comment:
which line is this error thrown from?

Reply:
https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/routes/audiusUsers.js#L16 !